incubator/istio/README.md

```diff
@@ -148,5 +148,5 @@ These addons can be selectively installed by setting `addons.<addon-name>.enable
 If you are running a Kubernetes 1.7+ and have the Initializers api enabled you may choose to enable the Initializer to be installed. See the [docs](https://kubernetes.io/docs/admin/extensible-admission-controllers/) on how to enable.
 
 ```console
-helm install --name istio . --namespace istio-system --set istio.install=true,initializer.enabled=true
+helm install --name my-release incubator/istio --namespace istio-system --set istio.install=true,initializer.enabled=true
```

Comment:
Can you also add the --devel flag until we get the chart version sorted out.

Reply:
Done